docs/guides/errors.md

```diff
@@ -0,0 +1,156 @@
+# Errors
+
+All the errors that Prism returns conform to the [RFC7807](https://tools.ietf.org/html/rfc7807). In a nutshell, this means that you'll always get a **JSON object** with the following properties:
+
+* `type` (string)
+* `title` (string)
+* `status` (number)
+* `detail` (string)
+
+This document enumerates all the possible errors returned by Prism and provides some further informations on how to solve them.
+
+## Routing errors
+
+This class of error is returned when Prism is trying to identify the right resource to use to respond to the provided HTTP Request.
+
+### NO_BASE_URL_ERROR
```

Comment:
Does this generate an automatic ID for us that we can link to? `#NO_BASE_URL_ERROR` or does docs make it all pretty like `#no-base-url-error`? 

If this does not work as expected, we can do:

```
### <a id="NO_BASE_URL_ERROR"></a>No Base URL Error
```

Reply:
It will generate an anchor link we can use.